lib/Doctrine/DBAL/Driver/PDOConnection.php

```diff
@@ -32,6 +31,9 @@
  */
 class PDOConnection extends PDO implements Connection, ServerInfoAwareConnection
 {
+    /** @var string */
+    private $lastInsertId = '0';
```

Comment:
This default value + doc block don't seem valid since you assign LastInsertId below.